Dockerfile

```diff
@@ -32,6 +32,7 @@ ENV NGINX_VERSION 4.3.0
 ENV CROMWELL_CHART_VERSION 0.2.220
 ENV CROWELL_ON_AZURE_CHART_VERSION 0.2.220
 ENV WDS_CHART_VERSION 0.3.0
+ENV HAIL_BATCH_CHART_VERSION 0.1.4
```

Comment:
We will likely iterate on the helm chart a bit more and bump the version -- Hail team has said there are fixes forthcoming.